packages/categorize/configure/src/design/categories/choice-preview.jsx

```diff
@@ -27,7 +27,7 @@ export class ChoicePreview extends React.Component {
       <div className={classNames(classes.choicePreview, className)}>
         {choice ? (
           <Choice alternateResponseIndex={alternateResponseIndex} category={category} choice={choice} choiceIndex={choiceIndex} onRemoveChoice={() => this.delete()}>
-            <HtmlAndMath html={choice?.content} />
+            <HtmlAndMath html={choice?.content} className={`${classes.breakWord}`} />
```

Comment:
Why do you need the `${...}`? Why isn't it 
className={classes.breakWord} ?

Reply:
@andreeapescar if we send it as classes.breakWord, it will send it as an object and we need the  name of the classname (string) as props